lib/iron_worker_ng/feature/python/merge_requirements.rb

```diff
@@ -0,0 +1,35 @@
+module IronWorkerNG
+  module Feature
+    module Python
+      module MergeRequirements
+        module InstanceMethods
+
+          def merge_requirements(path)
+            feature = IronWorkerNG::Feature::Base.new(self)
+            IronWorkerNG::Fetcher.fetch_to_file(feature.rebase(path)) do |requirements|
```

Comment:
@sunloverz what will happen if we do this:
```python
command = '-r' + requirements
merge_pip(command)
```

Will it simulate the following command? 
```
pip install -r requirements.txt
```
Then I think you will not need to parse the file and try to build installation commands.